src/euphorie/content/browser/surveygroup.py

```diff
@@ -36,17 +39,41 @@ def surveys(self):
         return templates
 
 
+class ISurveySourceSelectionSchema(Interface):
+    source_selection = SurveySourceSelectionField(
+        title=_("label_source_selection", default="Choose a source"),
+        required=False,
+    )
+
+
+@implementer(ISurveySourceSelectionSchema)
+@adapter(Interface)
+class SurveySourceSelectionAdaptedContext:
+    def __init__(self, context):
+        pass
```

Comment:
The code before was not storing this kind of information as well.
I agree, it might be interesting to store it.
But, in that case, the most natural thing becomes moving the fields either to a behavior or to the existing content type schema.

Reply:
This is the point where I added the breakpoint and nothing happened.  So the `__init__` does not seem to get called.  Maybe I tried on the wrong url.